Scripts/Python/Neighborhood.py

```diff
@@ -81,3 +91,42 @@ def OnFirstUpdate(self):
             
     def OnNotify(self,state,id,events):
         pass
+
+    def OnServerInitComplete(self):
+        dnitime = PtGetDniTime()
+        month = time.strftime('%m', time.gmtime(dnitime))
+        day = time.strftime('%d', time.gmtime(dnitime))
+        agevault = ptAgeVault()
+        if agevault:
+            ageSDL = agevault.getAgeSDL()
+            if ageSDL:
+                if int(month) == 1 and int(day) <= 3: #New Years
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/#inline-comments) says inline comments should be avoided.

Reply:
Removed.